package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "tensorboard",
+  "version": "1.14.0",
+  "description": "TensorFlow's visualization toolkit",
+  "private": true,
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "postinstall": "ngc -p angular-metadata.tsconfig.json",
+    "build": "bazel build //...",
+    "test": "ibazel test //..."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tensorflow/tensorboard.git"
+  },
+  "keywords": [
+    "ml",
+    "visualization"
+  ],
+  "author": "TensorFlow authors",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/tensorflow/tensorboard/issues"
+  },
+  "homepage": "https://github.com/tensorflow/tensorboard#readme",
+  "devDependencies": {
```

Comment:
So does this essentially replace js.bzl now?  This becomes the source of truth for JS dependency versioning?

Reply:
Yes! We would not need js.bzl/polymer.bzl and other bzl variants. A bazel_nodejs rule will fetch the dependency correctly for us.
https://github.com/bazelbuild/rules_nodejs#using-bazel-managed-dependencies